docs/build-build-with-polkadot.md

```diff
@@ -300,28 +299,40 @@ the latest resources. You should also keep up to date with the following links:
-and want to deploy to a live environment.
+### Edgeware
 
-Edgeware intends to at some point connect to Polkadot as a parachain that allows for smart
-contracts. At this point, the smart contracts would be able to interact with other pieces of the
-Polkadot ecosystem through [XCMP](learn-crosschain.md).
+[Edgeware][edgeware] is a permissionless smart contract platform. It is an option for smart contract developers and want to deploy to a live environment. Users can deploy both EVM and WASM smart contracts on Edgeware. Edgeware intends to connect to Polkadot and Kusama as a smart contract parachain.
 
-Edgeware general documentation can be found [here][edgeware documentation] and [how to deploy smart
-contracts on Edgeware here][edgeware contracts documentation].
+Try deploying a smart contract on Edgeware by following their [documentation][edgeware contracts documentation].
 
 ### Moonbeam
 
 [Moonbeam][] is another project that is planning to deploy to Polkadot as a parachain and will
-support smart contracts. Since Moonbeam uses [Frontier][], an interoperability layer with existing
-Ethereum tooling, it will support all applications that are written to target the EVM environment
-with little friction.
+support Ethereum compatible smart contracts. Since Moonbeam uses [Frontier][], an interoperability layer with existing Ethereum tooling, it will support all applications that are written to target the EVM environment with little friction.
 
-Try deploying a contract to Moonbeam by following their [documentation][moonbeam docs].
+[Moonriver](https://docs.moonbeam.network/networks/moonriver/), a companion network to Moonbeam, launched as a parachain on Kusama. Parachain functionality is live, and features are being incrementally released. The final phase of the launch will include EVM functionality and balance transfers.
+
+Try deploying a smart contract to Moonbeam by following their [documentation][moonbeam docs].
+
+### Astar
+
+[Astar](https://astar.network/) (formerly Plasm) is built on Substrate as a smart contract platform and intends to integrate with Polkadot as a parachain. [Shiden](https://shiden.astar.network/) is currently live on Kusama, and supports EVM, WASM, and Layer 2 solutions. Astar aims to house all layer 2 scaling solutions and support all layer 2 protocols through the [OVM](Optimistic Virtual Machine).
+
+Try deploying an Ethereum or ink! smart contract by following their [documentation](https://docs.astar.network/build/smart-contracts).
+
+### Acala
+
+[Acala](https://acala.network/) is a decentralized finance consortium and DeFi infrastructure chain delivering a set of protocols to serve as DeFi hub on Polkadot. [Karura](https://acala.network/karura), Acala's canary network, is live as a parachain on Kusama. Interested teams are now able to deploy DApps and smart contracts on Karura's platform. Acala is also implementing the [Acala EVM](https://wiki.acala.network/learn/acala-evm/why-acala-evm).
```

Comment:
- we could say `... serve as the Defi hub on Polkadot ...` since this is Acala's claim